src/pages/accessibilite-numerique/VousEtes.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <Layout class="vous-etes-page">
+
+    <nav aria-label="Breadcrumb" class="breadcrumb">
+        <ol>
+          <li>
+            <g-link to="/">Accueil</g-link>
+          </li>
+          <li>
+            <g-link to="/accessibilite-numerique/">Accessibilité</g-link>
+          </li>
+          <li aria-current="page">Vous êtes</li>
+        </ol>
+    </nav>
+
+    <div class="cover">
+      <div class="cover__container">
+        <div class="cover__subhead"><Accessibilite class="cover__subhead-icon" focusable="false" aria-hidden="true"/>Accessibilité</div>
+        <h1 class="text-highlight">Vous êtes</h1>
```

Comment:
Je me demande si c'est un bon titre de page, dans l'optique de partager la page ? Après ce n'est pas facile, je reconnais. Quelque chose comme "Les rôles clés de la mise en œuvre de l'accessibilité numérique", enfin c'est long... Mais le titre SEO peut-être un peu plus long, et sur la page une version plus courte "Les rôles clés", comme il y a le tag Accessibilité juste au dessus. Parce que je me dis, au delà des fiches métiers, c'est aussi une bonne vue d’ensemble de qui doit être impliqué et quel est son rôle à jouer.

Reply:
you are right